### PR TITLE
Create SunKid.java

### PR DESCRIPTION
**Pull request by AgentTroll**

### CODE REVIEW

File: src/main/java/io/mazenmc/skypebot/modules/generated/SunKid.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import com.skype.ChatMessage;
+import com.skype.SkypeException;
+import io.mazenmc.skypebot.engine.bot.Command;
+import io.mazenmc.skypebot.engine.bot.Module;
+import io.mazenmc.skypebot.utils.Resource;
+
+import java.util.*;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ChipDev implements Module {
```
Comment (mkotb) [inline src/main/java/io/mazenmc/skypebot/modules/generated/SunKid.java]: This won't compile

File: src/main/java/io/mazenmc/skypebot/modules/generated/SunKid.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            "Sorry did that upset you. Maybe you can draw yourself a picture",
+            "Mineplex, most likely won't be able to manual ban as they have 20k players", // Not sure wtf this was but ok
+            "I'm calling bullshit", 
+            "there is no way you made a full implementation of a bot" // there is more but it's funnier without it
+            "I'm sending actual key board input and mouse input",
+            "You would have to find something that I fucked up on that isn;t the norm",
+            "now that everyone is butt hurt and talking shit",
+            "Are you trolling?", // by far the best quote
+            "If you tp to someone and suspect them for bottling then you can ban them. Good luck detecting who;s bottling", // >bottling
+            I made the bot with 10 hours of free time anyone knowledgeable could re-create it",
```
Comment (nickrobson) [inline src/main/java/io/mazenmc/skypebot/modules/generated/SunKid.java]: Needs an open-quotes

Comment (AgentTroll) [inline src/main/java/io/mazenmc/skypebot/modules/generated/SunKid.java]: RIP :/

**Comment by AgentTroll** [comment]
Done

**Pull request merged**